index.html

```diff
@@ -97,222 +106,235 @@ <h2 class="supporting-text">I'am a Software Developer</h2>
+                        A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard
+                    </p>
+                    <div>
+                        <ul class="card-tags">
+                            <li><button class="tag blur">html</button></li>
+                            <li><button class="tag blur">bootstrap</button></li>
+                            <li><button class="tag blur">Ruby</button></li>
+                        </ul>
+                    </div>
+                    <button class="btn-max-width hide-btn">See Project</button>
+                </div>
+                <!-- Fifth -->
+                <div class="work-card">
+                    <h2 class="card-header">Professional Art <br> Printing Data</h2>
+                    <p class="card-description">
+                        A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard
+                    </p>
+                    <div>
+                        <ul class="card-tags">
+                            <li><button class="tag blur">html</button></li>
+                            <li><button class="tag blur">bootstrap</button></li>
+                            <li><button class="tag blur">Ruby</button></li>
+                        </ul>
+                    </div>
+                    <button class="btn-max-width hide-btn">See Project</button>
+                </div>
+                <!-- sixth -->
+                <div class="work-card">
+                    <h2 class="card-header">Professional Art <br> Printing Data</h2>
+                    <p class="card-description">
+                        A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard
+                    </p>
+                    <div>
+                        <ul class="card-tags">
+                            <li><button class="tag blur">html</button></li>
+                            <li><button class="tag blur">bootstrap</button></li>
+                            <li><button class="tag blur">Ruby</button></li>
+                        </ul>
+                    </div>
+                    <button class="btn-max-width hide-btn">See Project</button>
                 </div>
-                <button class="btn-max-width">See Project</button>
             </div>
         </div>
     </div>
     </section>
     <section class="about-section">
-        <div>
-            <h1  class="primary-text">about me</h1>
-        </div>
-        <div>
-            <h2 class="description">
-            Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.
-            </h2>
+        <div class="about-container">
+            <div>
```

Comment:
The About me section overlaps the rest of the recent works section on the desktop and the recent work section images and descriptions are not showing up in mobile view. See screenshots below:

- Desktop
<img width="911" alt="image" src="https://user-images.githubusercontent.com/78151836/208215250-f2cf9361-e30b-42b0-829e-36e88fb3bd90.png">

- Mobile

<img width="266" alt="image" src="https://user-images.githubusercontent.com/78151836/208215327-6e42cada-d669-4eff-824b-ecfe88a2b0d7.png">

Reply:
Updated

- position of about section to resolve overlapping divs
- added content and background to mobile version work cards.